src/components/popover-container/popover-container.stories.tsx

```diff
@@ -314,3 +314,15 @@ export const Filter: ComponentStory<typeof PopoverContainer> = () => {
     </Box>
   );
 };
+
+export const DisableAnimation: ComponentStory<typeof PopoverContainer> = () => (
+  <div style={{ height: 100 }}>
+    <PopoverContainer
+      disableAnimation
```

Comment:
**suggestion** (non-blocking): I'd personally add a `title` to this story - most of the others have one - as this should remove the need to add the `containerAriaLabel` and `openButtonAriaLabel` to stop Axe flagging accessibility errors, as well as more generally show the best practices. Also because the story is only meant to demonstrate the `disableAnimation` prop so any other props are just distraction - I personally find a `title` prop less distracting than those 2 aria props. Not a big issue though if you disagree.

Reply:
I agree with you. A `title` is a lot better approach than this. Thanks for pointing it out. God knows what I was thinking here 😆

Reply:
looks like you've still left the `openButtonAriaLabel`, while getting rid of the other one. Not a big deal but my idea was that they should both be able to go away, leaving only a `title` prop and the `disableAnimation`.